src/main/java/de/craften/plugins/rpgplus/scripting/api/BungeecordModule.java

```diff
@@ -0,0 +1,94 @@
+package de.craften.plugins.rpgplus.scripting.api;
+
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.messaging.PluginMessageListener;
+import org.luaj.vm2.LuaString;
+import org.luaj.vm2.LuaTable;
+import org.luaj.vm2.LuaValue;
+import org.luaj.vm2.Varargs;
+
+import com.google.common.collect.Iterables;
+import com.google.common.io.ByteArrayDataInput;
+import com.google.common.io.ByteArrayDataOutput;
+import com.google.common.io.ByteStreams;
+
+import de.craften.plugins.rpgplus.RpgPlus;
+import de.craften.plugins.rpgplus.scripting.ScriptingModule;
+import de.craften.plugins.rpgplus.scripting.util.luaify.LuaFunction;
+import de.craften.plugins.rpgplus.scripting.util.luaify.Luaify;
+
+public class BungeecordModule extends LuaTable implements ScriptingModule, PluginMessageListener{
+
+	private final RpgPlus plugin;
+	
+	public BungeecordModule(final RpgPlus plugin) {
+		
+		this.plugin = plugin;
+		
+		plugin.getServer().getMessenger().registerOutgoingPluginChannel(plugin, "BungeeCord");
+		plugin.getServer().getMessenger().registerIncomingPluginChannel(plugin, "BungeeCord", this);
+		
+		Luaify.convertInPlace(this);
+	}
+	
+	@Override
+	public void onPluginMessageReceived(String channel, Player player, byte[] message) {
+		if (!channel.equals("BungeeCord")) {
+		      return;
+		}
+		ByteArrayDataInput in = ByteStreams.newDataInput(message);
+		String subchannel = in.readUTF();
+		
```

Comment:
You receive the message and… do nothing with it? :smile: 
This would be a good place to allow event listeners to be registered so that messages can be received from scripts.